ArduPlane/servos.cpp

```diff
@@ -266,7 +269,7 @@ void Plane::dspoiler_update(void)
         if (flap_percent > 0) {
             float inner_flap_scaled = (float)flap_percent;
             float outer_flap_scaled = (float)flap_percent;
-            if (progresive_crow) {
+            if (progresive_crow || crow_mode == Plane::Crow_Mode::Progressive) {  //progressive crow when option is set or RC switch is set to progressive
```

Comment:
I think this if would be better on line 205,

const bool progresive_crow   = (bitmask & CrowFlapOptions::PROGRESSIVE_CROW) != 0 || crow_mode == Plane::Crow_Mode::Progressive

Reply:
done